src/estimation/include/iDynTree/Estimation/AttitudeEstimatorUtils.h

```diff
@@ -147,4 +147,35 @@ iDynTree::UnitQuaternion composeQuaternion2(const iDynTree::UnitQuaternion &q1,
  */
 iDynTree::UnitQuaternion pureQuaternion(const iDynTree::Vector3& bodyFixedFrameVelocityInInertialFrame);
 
+
+/**
+ * @brief exponential map for quaternion - maps angular velocities to quaternion
+ *
```

Comment:
Can you document the math of  method (for example with the formula that you put in the PR doc)? Thanks!

Reply:
Fixed in https://github.com/robotology/idyntree/pull/547/commits/2469a9645c39e166d3202f2f53522f67e481bce0